content/docs/command-reference/get.md

```diff
@@ -44,6 +44,10 @@ that DVC-tracked targets should be found in a
 [`.dvc`](/doc/user-guide/dvc-files-and-directories#dvc-files) file of the
 project.
 
+> Note that `dvc list` supports `path` granularity for directories nested inside
+> directories
+> [tracked as a whole](/doc/command-reference/add#example-directory).
```

Comment:
This `path granularity` might do more harm than good. It is a technical in-house term that we use as developers, but for users it is probably rather confusing. Hence why I was thinking about not using it at all, as being able to get particular files/directories is a natural expected thing.

Reply:
I agreed with you at first, but then I checked all the commands that accept targets and they don't all support granularity e.g. `repro` accepts tracked files/dirs as targets (see #1574 BTW) but not granularly i.e. files/dirs inside dirs tracked as a whole.

So maybe, if this is considered "a natural expected thing" then I can remove these notes and add a note in all the commands that don't support granularity to clarify so. I personally don't think it's natural and expected, but Idk.

Reply:
These notes should become a simple part of the `targets` description. No need to invent any complicated terms, specific notes, etc to my mind.

Reply:
Good idea. Done that in d67d0cac for the `path` ones (corresponding to this comment). Will try to update the `target` ones accordingly... but it may be trickier because we don't have a paragraph dedicated to explain the `targets` arg.